viime/models.py

```diff
@@ -311,18 +309,13 @@ def fix_file_name(self, data, **kwargs):
 
     @post_dump
     def read_csv_file(self, data, **kwargs):
-        def get_csv(key):
-            if data.get(key) is not None:
-                return data[key].to_csv()
-
         if 'table' not in data:
             return data
 
         data['table'] = data['table'].to_csv(header=False, index=False)
-        data['measurement_table'] = get_csv('measurement_table')
-        data['measurement_metadata'] = get_csv('measurement_metadata')
-        data['sample_metadata'] = get_csv('sample_metadata')
-        data['groups'] = get_csv('groups')
+        if data.get('measurement_table') is not None:
+            data['measurement_table'] = data['measurement_table'].to_dict(
+                orient='split')
```

Comment:
Any reason you aren't converting the main table as well?

Reply:
cause in the cleanup table the source table is many treated as "raw" file (e.g. no meaning of column headers, row headers, typing, ...). Thus, I wanted to keep it similar without converting